src/diffusers/configuration_utils.py

```diff
@@ -46,6 +50,10 @@
 
 _re_configuration_file = re.compile(r"config\.(.*)\.json")
 
+_all_available_pipeline_component_modules = (
+    list(model_modules.values()) + list(scheduler_modules.values()) + list(pipeline_modules.values())
+)
```

Comment:
```suggestion
_all_available_pipeline_component_modules = sum((
    list(model_modules.values()) + list(scheduler_modules.values()) + list(pipeline_modules.values())
), [])
```

We need to make sure that the list is flattened